### DISCUSSION TITLE
DaisyUI for web components (custom elements)

### DISCUSSION THREAD

**Discussion by RamAddict** [saadeghi/daisyui#1654] [Q&A] [answered]
So I'm building a web component and I'm using daisy UI + vite + svelte. I need to be able to change DaisyUI's rem definitions to em, so whoever uses the component doesn't have to be careful if they changed the html font-size. Another issue is that Daisy sets the video, img, svg and other tags globally to display block, so whoever has these styles in the head will suffer interference. I think I might be able to do fix this if there was a pre-bundled version of daisyUI, but I'm still unsure about how to deal with this issue. Any help appreciated!

**Comment by saadeghi** [comment, marked as answer]
> I need to be able to change DaisyUI's rem definitions to em

I think you can do that using PostCSS plugins. Like this one: https://www.npmjs.com/package/postcss-replace

> Daisy sets the video, img, svg and other tags globally to display block  

That style is not from daisyUI. It's part of Tailwind CSS' preflight reset style: https://tailwindcss.com/docs/preflight
